ItemLibrary/Item.cs

```diff
@@ -37,10 +39,13 @@ public abstract class Item
 
         [MaxLength(256)]
         public string ImageLink { get; set; }
-        public ItemCategory ItemCategory { get; set; }
+        public Categories.ItemCategory ItemCategory { get; set; }
```

Comment:
i believe the code would be more readable if you would write 
using static ModelLibrary.Categories;
 public ItemCategory ItemCategory { get; set; }
 
 instead of 
 public Categories.ItemCategory ItemCategory { get; set; }

Reply:
Didn't see it. I guess ReSharper changed it and I didn't notice. Done.